modules/core/02-client/migrations/v7/genesis.go

```diff
@@ -0,0 +1,82 @@
+package v7
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/cosmos/ibc-go/v6/modules/core/02-client/types"
```

Comment:
should we alias this as `clienttypes`? I think having a `types` import without the alias adds a little overhead in reading the code. (more so if other types imorts exist which is not the case here!)

Reply:
Yea I agree. The reasoning for why it is types is because it is within the `02-client` subdirectory, but it is a little removed from that so I think it makes sense to use `clienttypes`